src/components/Dashboard.vue

```diff
@@ -100,7 +129,18 @@ export default {
       isOpeningRequirements: false,
       isTablet: window.innerWidth <= 878,
       isMobile: window.innerWidth <= 440,
-      maxBottomBarTabs: window.innerWidth <= 1347 ? 2 : 4
+      maxBottomBarTabs: window.innerWidth <= 1347 ? 2 : 4,
+      welcome: 'Welcome Cornellian!',
+      welcomeBodytext: 'View your college requirements, plan your semesters and courses, and more.',
+      welcomeExit: 'No, I want to skip this',
+      welcomeButtonText: 'Start Tutorial',
+      welcomeHidden: false,
+      startTour: false,
+      showTourEndWindow: false,
+      congrats: 'Congratulations! That’s a wrap',
+      congratsBodytext: 'Other than this, there is more you can explore, so feel free to surf through CoursePlan 🏄',
```

Comment:
Instead of putting this emoji in the text, can you save it as an image and use that instead? Let me know if you need help.